docs/utilities/data_classes.md

```diff
@@ -263,7 +263,7 @@ See also [this blog post](https://aws.amazon.com/blogs/compute/introducing-iam-a
 
     @event_source(data_class=APIGatewayAuthorizerEventV2)
     def handler(event: APIGatewayAuthorizerEventV2, context):
-        user = get_user_by_token(event.get_header_value("x-token"))
+        user = get_user_by_token(event.headers["x-token"])
```

Comment:
How about using `event.headers.get()`? So we prevent the customer from having problems with `KeyError`.